docs/_sources/new_bandit.rst.txt

```diff
@@ -113,7 +113,7 @@ Here is, at a high-level, what you need to implement in your bandit policy:
             # These fields are declared here and initialized to zero.
             self.my_value_to_arm = dict.fromkeys(self.arms, 0)
 
-        def fit(self, decisions: np.ndarray, rewards: np.ndarray, contexts: np.ndarray = None) -> NoReturn:
+        def fit(self, decisions: np.ndarray, rewards: np.ndarray, contexts: np.ndarray = None):
```

Comment:
May I suggest that, we return a "None" instead of the incorrect NoReturn 

from () -> NoReturn to () -> None

This is clearly a miss on our part, so we highly appreciate your fix!